### PR TITLE
2.7.2 errata https://issues.redhat.com/browse/ACM-4269

### PR DESCRIPTION
**Pull request by oafischer**
https://issues.redhat.com/browse/ACM-4269

### CODE REVIEW

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -7,6 +7,12 @@ By default, Errata updates are automatically applied when released. See link:../
 
 FIPS notice: If you do not specify your own ciphers in `spec.ingress.sslCiphers`, then the `multiclusterhub-operator` provides a default list of ciphers. For 2.4, this list includes two ciphers that are _not_ FIPS approved. If you upgrade from a version 2.4.x or earlier and want FIPS compliance, remove the following two ciphers from the `multiclusterhub` resource: `ECDHE-ECDSA-CHACHA20-POLY1305` and `ECDHE-RSA-CHACHA20-POLY1305`.
 
+== Errata 2.7.2
+
+* You can now use {ocp} 4.12 on Microsoft Azure. (https://issues.redhat.com/browse/ACM-3223[ACM-3223])
```
Comment (dockerymick) [inline release_notes/errata.adoc]: I recommend that you say:

Adds support for the use of {ocp} 4.12 on Microsoft Azure. 

For consistency with syntax, please add `link:` before the URLs ([Linking guidance](https://stolostron.github.io/rhacm-docs/wr_linking))

For example:

`link:https://issues.redhat.com/browse/ACM-3223[ACM-3223]`

**Review by dockerymick — CHANGES_REQUESTED** [review]
Made 2 suggestions

File: release_notes/errata.adoc (hunk 1 above)
Comment (oafischer) [reply release_notes/errata.adoc]: Fixed, thanks!

**Review by dockerymick — APPROVED** [review]
Looks good to me!

**Pull request merged**